tests/integration/data/regression_output/linux/stacks -t bogus

```diff
@@ -1 +1 @@
-sdb: stacks: 'bogus' is not a valid task state (acceptable states: RUNNING, INTERRUPTIBLE, UNINTERRUPTIBLE, STOPPED, TRACED, DEAD, ZOMBIE, PARKED, IDLE)
+sdb: stacks: 'BOGUS' is not a valid task state (acceptable states: RUNNING, INTERRUPTIBLE, UNINTERRUPTIBLE, STOPPED, TRACED, DEAD, ZOMBIE, PARKED, IDLE)
```

Comment:
this diff is because the argument passed by the  user was mutated (see other comment)